webpack.plugins.config.js

```diff
@@ -0,0 +1,53 @@
+var webpack = require('webpack');
```

Comment:
Currently, the other `webpack.*.config.js` files follow the pattern of exporting webpack config objects - common.config is the base, and the others extend it. This one's different - it's exporting helper functions that return pieces of config.

Let's create a `webpack` directory in `buildtools/`, move this there, and rename it to something like `plugins.js`. In general, there are a bunch of things we can do to clean these webpack files, and this is a good start.